### PR TITLE
Added test for sample script update_domain.sh ( wlst )

### PR DESCRIPTION
**Pull request by hzhao-github**
3 failures from Jenkins job https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5942 are not caused by this PR

### CODE REVIEW

**Review by anpanigr — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 1]
```diff
@@ -70,7 +70,7 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @DisplayName("Verify the domain on pv, domain in image samples using wlst and wdt and domain lifecycle scripts")
 @IntegrationTest
-class ItWlsSamples {
+public class ItWlsSamples {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:73]: Can you please remove the public from here and also all the test methods?
Ryan merged https://github.com/oracle/weblogic-kubernetes-operator/pull/2492 to make this change and you are overwriting this change.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 2]
```diff
@@ -155,7 +155,7 @@ public static void initAll(@Namespaces(5) List<String> namespaces) {
   @ParameterizedTest
   @MethodSource("paramProvider")
   @DisplayName("Test samples using domain in image")
-  void testSampleDomainInImage(String model) {
+  public void testSampleDomainInImage(String model) {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:158]: Remove public - see earlier comment.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 3]
```diff
@@ -217,7 +217,7 @@ void testSampleDomainInImage(String model) {
   @ParameterizedTest
   @MethodSource("paramProvider")
   @DisplayName("Test samples using domain in pv")
-  void testSampleDomainInPv(String model) {
+  public void testSampleDomainInPv(String model) {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:220]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 4]
```diff
@@ -268,7 +267,7 @@ void testSampleDomainInPv(String model) {
   @Order(3)
   @Test
   @DisplayName("Test server lifecycle samples scripts")
-  void testServerLifecycleScripts() {
+  public void testServerLifecycleScripts() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:270]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 5]
```diff
@@ -292,7 +291,7 @@ void testServerLifecycleScripts() {
   @Order(4)
   @Test
   @DisplayName("Test server lifecycle samples scripts with constant replica count")
-  void testServerLifecycleScriptsWithConstantReplicaCount() {
+  public void testServerLifecycleScriptsWithConstantReplicaCount() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:294]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 6]
```diff
@@ -318,7 +317,7 @@ void testServerLifecycleScriptsWithConstantReplicaCount() {
   @Order(5)
   @Test
   @DisplayName("Test cluster lifecycle scripts")
-  void testClusterLifecycleScripts() {
+  public void testClusterLifecycleScripts() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:320]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 7]
```diff
@@ -339,7 +338,7 @@ void testClusterLifecycleScripts() {
   @Order(6)
   @Test
   @DisplayName("Test domain lifecycle scripts")
-  void testDomainLifecycleScripts() {
+  public void testDomainLifecycleScripts() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:341]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 8]
```diff
@@ -363,7 +362,7 @@ void testDomainLifecycleScripts() {
   @Order(7)
   @Test
   @DisplayName("Manage Traefik Ingress Controller with setupLoadBalancer")
-  void testTraefikIngressController() {
+  public void testTraefikIngressController() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:365]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 9]
```diff
@@ -376,7 +375,7 @@ void testTraefikIngressController() {
   @Order(8)
   @Test
   @DisplayName("Manage Voyager Ingress Controller with setupLoadBalancer")
-  void testVoyagerIngressController() {
+  public void testVoyagerIngressController() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:378]: Remove public

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java [hunk 10]
```diff
@@ -389,7 +388,7 @@ void testVoyagerIngressController() {
   @Order(9)
   @Test
   @DisplayName("Manage Nginx Ingress Controller with setupLoadBalancer")
-  void testNginxIngressController() {
+  public void testNginxIngressController() {
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java:391]: Remove public

**Review by bhavaniravichandran — CHANGES_REQUESTED** [review]

**Comment by robertpatrick** [comment]
What is the status of this PR? It is over 4 months old so is it still relevant? If so, then update the PR to make it so that it can be merged with current code. Otherwise, please close it.

**Pull request closed without merging**